convert2rhel/rhelvariant.py

```diff
@@ -33,6 +38,15 @@ def determine_rhel_variant():
     if not tool_opts.variant:
         tool_opts.variant = _user_to_choose_rhel_variant()
     loggerinst.info("Variant: %s" % tool_opts.variant)
+
+    # checking RHN Classic
+    loggerinst.info("Checking RHN Classic")
+    if _check_rhn_classic():
+        loggerinst.critical("RHN Classic detected! \n"
+                            "Please follow the bellow link for more information how to disable it:\n"
+                            "https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html/migrating_from_rhn_classic/index"  # pylint: disable=C0321
+                            "\n\n Convertion will not proceed.")
+
     return
 
 
```

Comment:
```suggestion
rhn_reg_file = utils.RestorableFile(_RHN_REGISTRATION_FILE)  # pylint: disable=C0103
```

Also, add `rhelvariant.rhn_reg_file.backup()` to https://github.com/oamg/convert2rhel/blob/master/convert2rhel/main.py#L72.
And add `rhelvariant.rhn_reg_file.restore()` to https://github.com/oamg/convert2rhel/blob/master/convert2rhel/main.py#L212.

In another PR, we need to refactor how we do these backups and restores.